frontend/src/app/components/application-banner/application-banner.component.html

```diff
@@ -22,15 +22,12 @@
             id="header-accordion"
             class="content-center"
           >
-            <mat-expansion-panel-header
-              class="d-flex justify-content-center gap-2 mx-3"
-              [attr.data-testId]="'expansion-panel-header'"
-            >
+            <mat-expansion-panel-header class="d-flex justify-content-center gap-2 mx-3" [attr.data-testId]="'expansion-panel-header'">
```

Comment:
What is the logic why this is on a single line and [this](https://github.com/puzzle/okr/pull/1221/files#diff-dc0372338d24a2f21dc738836a6e38e025205198febac14658a6fa9c2edc44f9R3), as an example, not?

Reply:
as discussed